docs/streams/getting-started-streams.md

```diff
@@ -11,12 +11,15 @@ This article is an introduction to using streams with [Macrometa SDKs](../sdks/i
 ## Prerequisites
 
 - Access to a Macrometa account with sufficient permissions to create streams.
+- Have your API Key ready. For more information, refer to [Create API Keys] (../account-management/api-keys/create-api-keys)
 - Install the appropriate SDK. For more information, refer to [Install SDKs](../sdks/install-sdks.md).
 
 ## Get Started with Streams
 
 The basic parts of this quickstart walk you through creating a stream, publishing messages to it, and subscribing to the stream using the [pyC8](https://pyc8.readthedocs.io/en/latest/) and [jsC8](https://www.npmjs.com/package/jsc8) SDKs.
 
+To get started, create a new javascript (.js) or python (.py) file in your favorite IDE. Then, copy the code block below and paste it into your javascript or python file. With each subsequent step, append the code block to the existing file and run it.
```

Comment:
```suggestion
To get started, create a new JavaScript (.js) or python (.py) file in your favorite IDE. Then, copy the code block below and paste it into your JavaScript or Python file. With each subsequent step, append the code block to the existing file and run it.
```

Reply:
Looks good! We forgot to capitalize the first instance of the word 'python'